test/Common.FunctionalTests/ClientCertificateTests.cs

```diff
@@ -54,13 +57,14 @@ private async Task ClientCertTest(TestVariant variant, bool sendClientCert)
             deploymentParameters.ApplicationBaseUriHint = $"https://localhost:{port}/";
             deploymentParameters.AddHttpsToServerConfig();
 
-            var deploymentResult = await DeployAsync(deploymentParameters);
             var handler = new HttpClientHandler
             {
                 ServerCertificateCustomValidationCallback = (a, b, c, d) => true,
                 ClientCertificateOptions = ClientCertificateOption.Manual,
             };
 
+            var deploymentResult = await DeployAsync(deploymentParameters);
```

Comment:
What did this change?

Reply:
Sorry, meant to put this even lower. This is to make it so w3wp is create after creating the client handler.